mkdocs/docs/protocol/non-merklized.md

```diff
@@ -0,0 +1,126 @@
+
+**Core Concept**
+
+Non merklized credentials now can be created relying on JSON-LD schemas.
+
+Example of such context: `https://schema.iden3.io/core/jsonld/auth.jsonld`
+
+```json
+{
+  "@context": [{
+    "@version": 1.1,
+    "@protected": true,
+    "id": "@id",
+    "type": "@type",
+    "AuthBJJCredential": {
+      "@id": "https://schema.iden3.io/core/jsonld/auth.jsonld#AuthBJJCredential",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "iden3_serialization": "iden3:v1:slotIndexA=x&slotIndexB=y",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "auth-vocab": "https://schema.iden3.io/core/vocab/auth.md#",
+        "x": {
+          "@id": "auth-vocab:x",
+          "@type": "xsd:positiveInteger"
+        },
+        "y": {
+          "@id": "auth-vocab:y",
+          "@type": "xsd:positiveInteger"
+        }
+      }
+    }
+  }]
+}
+```
+
+Schema is defined as a schema for non-merklized credentials by utilizing `iden3_serialization` attribute.
+
+It contains a map string represented mapping, where:
+
+`iden3:v1` - version of protocol serialization, constant.
+
+`slotIndexA=x`  slotIndexA is index data slot  A with index 2 for path to field  `x` in credential
+
+`slotIndexB=y`  slotIndexB is index data slot B  with index 3 for path to field `y` in credential
+
+other possible values:
+
+`slotValueA`  slotIndexB is value data slot  with index 6
+
+`slotValueB`  slotValueB is value data slot  with index 7
+
+See more information regarding data slot indexes [here](https://www.notion.so/Identity-Core-77fe2f04c8ad4e0296e2b90400d7ae3a?pvs=21). When user creates schema he should choose the slot to put the field.
+
+Nested structures are supported and path are created using concatenation with `.` e.g for birthday field in the Index Data Slot A mapping entry looks like that: `"...slotIndexA=passportInfo.birthday"`
+
+```json
+{
+  ...
+			...
+          "passportInfo": {
+            "@id": "vocab:passportInfo",
+            "@context": {
+              "@version": 1.1,
+              "@protected": true,
+              "kyc-vocab": "https://github.com/iden3/claim-schema-vocab/blob/main/credentials/kyc.md#",
+              "xsd": "http://www.w3.org/2001/XMLSchema#",
+              "id": "@id",
+              "type": "@type",
+              "birthday": {
+                "@type": "xsd:integer",
+                "@id": "vocab:birthday"
+              }
+            }
+          }
+      ...
+  ...
+}
+```
```

Comment:
Would be good to have full json-ld schema here as another example of NMC schema.

Reply:
fixed